lib/index.d.ts

```diff
@@ -1730,6 +1730,7 @@ declare namespace sharp {
 
     type Matrix2x2 = [[number, number], [number, number]];
```

Comment:
Not sure `Matrix2x2` is used anywhere, I suspect this was inherited via DefinitelyTyped and could be removed.

Reply:
this type are used at affine() //2x2 affine transformation matrix